commands/games/roulette-info.js

```diff
@@ -0,0 +1,39 @@
+const { Command } = require('discord.js-commando');
+const stripIndents = require('common-tags').stripIndents;
+
+module.exports = class RouletteInfo extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'roulette-info',
+			group: 'games',
+			memberName: 'roulette-info',
+			description: 'Displays information about the roulette.',
+			details: 'Displays information about the roulette.'
+		});
+	}
+
+	async run(msg) {
+		return msg.embed({
+			image: { url: 'http://i.imgur.com/l88QEHm.png' },
```

Comment:
Should probably use HTTPS and use an imagehost where I know it wont run out, use cuntflaps.me if possible 👌